docs/reference/operations-guide/core.md

```diff
@@ -102,6 +102,68 @@ Adds elements to a graph. [Javadoc](https://gchq.github.io/Gaffer/uk/gov/gchq/ga
       5 --> 6
     ```
 
+## DeleteElements
+
+Deletes Elements from a graph. [Javadoc](....)
```

Comment:
Could put this as a link to the package the operation will be in for now can always update before releasing the 2.3 docs